com.unity.render-pipelines.universal/Runtime/Data/PostProcessData.cs

```diff
@@ -11,7 +11,7 @@ namespace UnityEngine.Rendering.Universal
     /// </summary>
     /// <seealso cref="Shader"/>
     /// <seealso cref="Texture"/>
-    [Serializable]
+    [Serializable, ReloadGroup]
```

Comment:
@ellioman @lukas-unity This was reverted in another PR because it caused failures in our test. But I think I don't fully why it was reverted as this seems necessary to proper reload resources. 
 
 Is it because of the way we strip post processing? Maybe we need to look at it together.

Reply:
IIRC the problem was that reload group in case of finding reference as null, will allocate new one with default constructor. Which does not work well in case reference is MonoBehaviour or SerializedObject as they need to be serialized into asset. During build this resulted in somewhat corrupted state, where reference was set, but it was poiting to nothing (As newly created class was not serialized into asset).
Tests were failing with:
Expected null
Actual <null>

Reply:
@ellioman can eleborate more on it. If this is still the case, I would simply fix this in reload group. Basically stoping recursive loading if class is null and type of monobehviour or serialized object. As seems like all alternative solutions are unstable and keeps breaking again and again.

Reply:
Yes, @lukaschod explained the issue. The test we have to make sure we do not include PostProcessing in any other projects than PostPro was failing because of that.
When we run our CI on one of the URP Test projects besides PostPro, we will quickly see if this is still working or not.